dockerfiles/Dockerfile.cross

```diff
@@ -1,4 +1,4 @@
-FROM	dockercore/golang-cross:1.12.2@sha256:ea93d7ed5b464e5163cf8df40a198ad54afe6a59e1ca335c9bc4a5ed3702f2d0
+FROM	dockercore/golang-cross:1.12.3@sha256:b88a6be2469d321601f87675c49d1b3f166c4c37abc5ce3e74557fc6660481be
```

Comment:
as @thaJeztah  [said](https://github.com/docker/cli/pull/1816#discussion_r273193035), 

> I'm actually thinking of removing this "pin by digest", as it doesn't add much for our use-case :thinking: 

I also agree very much. Do we need to remove this "pin by digest" ? :thinking:

Reply:
I am ok with removing it

@vdemeester @silvin-lubecki SGTY?